src/ssl.c

```diff
@@ -2829,6 +2829,60 @@ const WOLFSSL_EVP_CIPHER *wolfSSL_EVP_get_cipherbyname(const char *name)
     return NULL;
 }
 
+const WOLFSSL_EVP_CIPHER *wolfSSL_EVP_get_cipherbynid(int id)
```

Comment:
That is close, something so that other developers can quickly get a summary of what the function does and it can eventually be automatically parsed. A good example to look at for the comments is the function wolfSSL_write_dup in src/ssl.c.